src/commcare_cloud/ansible/roles/elasticsearch/tasks/main.yml

```
@@ -1,18 +1,45 @@
+- name: "Check if Cluster Name is updated for Elasticsearch V7"
+  assert:
+    that:
+      - "'es7_' in elasticsearch_cluster_name"
+    fail_msg: "Elasticsearch V7 cluster name should start with es7_ prefix"
+  when: elasticsearch_version  == "7.9.1"
```

what about checking for > v7?
```suggestion
  when: elasticsearch_version is version('7.0.0', '>=')
```